### PR TITLE
bash-completions

### LINKED ISSUE #51: [feature request] command completion
Hi.

If possible, it would be useful to allow command completion to `gobrew`. This feature is present in many environment managers. Just as an example, by typing `gobrew in`, it would auto complete to `gobrew install <cursor-here>` by hit in TAB, and another hit in TAB would list the available versions.

### PR DESCRIPTION
**Pull request by kevincobain2000**
- [x] Fixes #51 
- [x] Update README
- [x] Won't work on `ls-remote`

### CODE REVIEW

File: gobrew.go [hunk 1]
```diff
@@ -152,7 +152,7 @@ func (gb *GoBrew) ListVersions() error {
 			version = cv + "*"
 			utils.Successln(version)
 		} else {
-			log.Println(version)
```
Comment (juev) [inline gobrew.go:155]: why you changed log to fmt?

Comment (kevincobain2000) [reply gobrew.go:155]: Stdout is simpler to pipe instead or stderr. I can change it back. Stdout makes more sense here I thought

Comment (juev) [reply gobrew.go:155]: What we have?

Снимок экрана 2023-01-13 в 10 07 08

fmt in 176 line do not have line ending. We should added it for getting the same result.

One more, I think we should use one library for output on all cases. fmt/color/etc

But we shouldn't use different libraries for output.

Its my opinion.

For discussion I create #87

Comment (kevincobain2000) [reply gobrew.go:155]: Good catch

Comment (kevincobain2000) [reply gobrew.go:155]: About your second opinion using same lib, let me work on it.
Probably in same pull req or a new one. But in short, I agree.

**Pull request merged**